website/src/data/users.js

```diff
@@ -707,6 +707,14 @@ const Users = [
     source: null,
     tags: ["product"],
   },
+  {
+    title: 'Plausible Analytics Docs',
+    description: 'Plausible Analytics is a simple, open source, lightweight (< 1 KB) and privacy-friendly alternative to Google Analytics. ',
+    preview: require('./showcase/plausible.png'),
```

Comment:
@metmarkosaric it looks like you forgot to commit screenshot of the site. Can you fix that?